src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity.php

```diff
@@ -502,6 +495,25 @@ public static function fromManageResponse(
         return $entity;
     }
 
+    private static function setRedirectUrisFromManageResponse(
+        Entity $entity,
+        ManageEntity $manageEntity,
+        $environment,
+        $playGroundUriTest = '',
+        $playGroundUriProd = ''
+    ) {
+        $redirectUris = $manageEntity->getOidcClient()->getRedirectUris();
+        $playGroundUri = ($environment == self::ENVIRONMENT_PRODUCTION ? $playGroundUriProd : $playGroundUriTest);
```

Comment:
If the playgroundUriProd is set to '', we now default to playgroundTest. Is that what we want? In other words, what's the meaning of the default values for the playGroundUriProd and playGroundUriTest?

Reply:
See comment above.